be/src/column/binary_column.cpp

```diff
@@ -62,6 +63,24 @@ void BinaryColumnBase<T>::append(const Column& src, size_t offset, size_t count)
 
 template <typename T>
 void BinaryColumnBase<T>::append_selective(const Column& src, const uint32_t* indexes, uint32_t from, uint32_t size) {
+    if (src.is_constant()) {
```

Comment:
it seems nullable column, const null column, ordinary column processing branches are missing!
and constant column dictation is illegal, use statment as follows!!
if(!src.is_only_null() && src.is_constant()) {
...
}

Reply:
fixed